udata/api/oauth2.py

```diff
@@ -317,7 +317,7 @@ def save_token(token, request):
         user = request.user or client.owner
         OAuth2Token.objects.create(
             client=client,
-            user=user,
+            user=user.id,
```

Comment:
Are you sure? the OAuth2Token.user is a reference.

Reply:
Yes, but the ID is always available in MongoEngine on objects. This avoid testing `isinstance(user, LocalProxy)` and dereference if necessary (was the case during test fro implicit grant)